### ISSUE TITLE
package: guitarix

### ISSUE THREAD

**Issue by atsushieno** [atsushieno/aap-lv2#2]
https://github.com/brummer10/guitarix/

Haven't investigated anything yet.

**Comment by atsushieno** [comment]
It depends on fftw3 which is not straightforward on Android.

**Comment by atsushieno** [comment]
guitarix needs zita-convolver too, and possibly more if we do not manually disable libgxwmm, pygxw, and glade-gxw.

**Comment by atsushieno** [comment]
Looks like glibmm is still required.

> ../src/LV2/gx_amp.lv2/gxamp.cpp:20:10: fatal error: 'glibmm.h' file not found
> #include <glibmm.h>

**Comment by atsushieno** [comment]
It built at android-native-audio-builders https://github.com/atsushieno/android-native-audio-builders/actions/runs/142831062

But we don't want to extract it with the current build script, as it will result in bloat glibmm dependencies just for guitarix. It had better be split.

**Comment by atsushieno** [comment]
https://github.com/atsushieno/android-native-audio-builders/issues/5

**Comment by atsushieno** [comment]
The Kotlin app is now merged into master (7fb5f34). But it still crashes when aaphostsample tries to use it:

```
   java.lang.UnsatisfiedLinkError: No implementation found for void org.androidaudioplugin.lv2.AudioPluginLV2ServiceExtension.initialize(java.lang.String, android.content.res.AssetManager) (tried Java_org_androidaudioplugin_lv2_AudioPluginLV2ServiceExtension_initialize and Java_org_androidaudioplugin_lv2_AudioPluginLV2ServiceExtension_initialize__Ljava_lang_String_2Landroid_content_res_AssetManager_2)
        at org.androidaudioplugin.lv2.AudioPluginLV2ServiceExtension.initialize(Native Method)
 ```

**Comment by atsushieno** [comment]
The UnsatisfiedLinkError was fixed. The next issue is that aaphostsample expects that there are two audio channels (right/left) for input/output respectively, while guitarix has only one input/output, and thus crashes.

**Comment by atsushieno** [comment]
Well, packaging is done. This issue should be closed.
